src/components/side_panel/chart/line_chart/line_chart_design_panel.xml

```diff
@@ -5,17 +5,12 @@
         <div class="o-section-title">Background color</div>
         <div class="o-with-color-picker">
           Select a color...
-          <span
-            t-att-style="props.definition.background ? `border-color: ${props.definition.background}` : 'border-bottom-style: hidden'"
-            t-on-click.stop="toggleColorPicker">
-            <t t-call="o-spreadsheet-Icon.FILL_COLOR"/>
-          </span>
-          <ColorPicker
-            t-if="state.fillColorTool"
-            onColorPicked="(color) => this.updateBackgroundColor(color)"
-            currentColor="props.definition.background"
-            t-key="backgroundColor"
-          />
+          <t t-call="o-spreadsheet-FillColorPickerEntry">
+            <t t-set="currentColor" t-value="props.definition.background"/>
+            <t t-set="onClick" t-value="toggleColorPicker"/>
+            <t t-set="showPicker" t-value="state.fillColorTool"/>
+            <t t-set="onColorPicked" t-value="(color) => this.updateBackgroundColor(color)"/>
+          </t>
```

Comment:
While using a simple template works, using a dedicated component is almost always a better option IMHO. Some advantages I see right of the bat:

- readability (`<FillColorEntry >` vs `<t t-call="o-..-FillColorEntry>`, `onClick="toggleColorPicker"` vs `<t t-set="onClick" t-value="toggle"`)
- props validation, we garantee in the tests that we aren't just sending some junk variables, and that all the needed props are given
- can define css inside the component